deploy/dynamo/sdk/src/dynamo/sdk/cli/serve_dynamo.py

```diff
@@ -91,19 +89,6 @@ class GracefulExit(SystemExit):
     pass
 
 
-def setup_signal_handlers():
-    """Setup signal handlers for graceful shutdown."""
-
-    def signal_handler(sig, frame):
-        logger.info(f"Received signal {sig}, initiating graceful shutdown")
-        raise GracefulExit(0)
-
-    # Register SIGINT and SIGTERM handlers
-    signal.signal(signal.SIGINT, signal_handler)
-    signal.signal(signal.SIGTERM, signal_handler)
-    signal.signal(signal.SIGQUIT, signal_handler)
```

Comment:
I'm basically unable to interactively kill or exit out of `dynamo serve` for TRTLLM example on latest `main` (I believe since @biswapanda 's part 1/N refactor changes), even my SIGQUITs are being caught, and never exiting:
![image](https://github.com/user-attachments/assets/b7e29d45-15ae-4115-b51d-683647448ac2)

Is this why you're removing the signal handlers?